pkg/scheduler/scheduler.go

```diff
@@ -318,8 +318,24 @@ func (e entryOrdering) Less(i, j int) bool {
 	if aBorrows != bBorrows {
 		return !aBorrows
 	}
+
 	// 2. FIFO.
-	return a.Obj.CreationTimestamp.Before(&b.Obj.CreationTimestamp)
+	evictedConditionPositiona := workload.FindConditionIndex(&a.Obj.Status, kueue.WorkloadEvicted)
```

Comment:
add a function that obtains the timestamp for each job to be used for the comparison